app/views/research_sessions/focus.html.erb

```diff
@@ -1,17 +1,17 @@
 <%= render 'back_link' %>
 <div class="main-content">
   <%= render 'error_summary' %>
-  <h1 class="visually-hidden">Research Focus</h2>
```

Comment:
Edit _textarea.scss to update the hint class to

```
.textarea__hint {
  color: $secondary-text;
  display: block;
  font-family: $font-family;
  font-size: $base-font-size;
  line-height: 24px;
  margin-top: $gutter / 2;
}
```

May need to do same for other form controls